homeassistant/components/starline/.translations/ru.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
This file needs to be removed. Translations, except the default english one, will be handled by Lokalise.